mbf_abstract_nav/include/mbf_abstract_nav/abstract_navigation_server.h

```diff
@@ -41,28 +41,28 @@
 #ifndef MBF_ABSTRACT_NAV__ABSTRACT_NAVIGATION_SERVER_H_
 #define MBF_ABSTRACT_NAV__ABSTRACT_NAVIGATION_SERVER_H_
 
-#include "abstract_planner_execution.h"
-#include "abstract_controller_execution.h"
-#include "abstract_recovery_execution.h"
```

Comment:
The style here should be: local includes in "" and installed includes in <>

Reply:
both are fine, but usually <> enforces a better style since it would fail to find just "abstract_planner_exectution" without the mbf_abstract_nav include path (and including the path has many benefits). but you decide

Reply:
I prefer "" here. My argumentation is based on: https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html 
I see your point with the namespace / path to include, but that should be clear here: Someone would directly see that these includes are local.